examples/subplots_axes_and_figures/mosaic.py

```diff
@@ -1,14 +1,10 @@
 """
+.. redirect-from:: /gallery/subplots_axes_and_figures/mosaic
```

Comment:
I think you want: 

```suggestion
.. redirect-from:: /tutorials/provisional/mosaic
```

Reply:
sigh, the dangers of grep + xargs + sed 🤦🏻